src/debug_utils.h

```diff
@@ -19,6 +19,27 @@
 #define COLD_NOINLINE
 #endif
 
+#if defined(__linux__) || defined(__sun)
+#include <link.h>
+#endif  // (__linux__) || defined(__sun)
+
+#ifdef __APPLE__
+#include <mach-o/dyld.h>  // _dyld_get_image_name()
+#endif  // __APPLE__
+
+#ifdef _AIX
+#include <sys/ldr.h>  // ld_info structure
+#endif  // _AIX
+
+#ifdef _WIN32
+#include <Windows.h>
+#include <process.h>
+#include <dbghelp.h>
+#include <Lm.h>
+#include <tchar.h>
+#include <psapi.h>
+#endif  // _WIN32
```

Comment:
Can you add these headers only to the `.cc` file?

Reply:
@addaleax - done, moved those to .cc, and performed `make format-cpp` that I missed last time.